packages/asc-ui/src/components/SearchBar/SearchBar.tsx

```diff
@@ -18,6 +18,7 @@ interface SearchBarProps {
   onFocus?: Function
   onKeyDown?: Function
   text?: string
+  upperRef?: any
```

Comment:
plz no any... Could be null | HTMLInputElement or HTMLElement or sth like that I think